backend/scripts/ci-load-test.js

```
@@ -276,19 +282,41 @@ function printSummary(config, campaigns, result) {
   console.log(`- Avg req/sec: ${formatFixed(result.requests.average)}`);
   console.log(`- Avg throughput: ${formatFixed(result.throughput.average / 1024)} KiB/s`);
 
-  console.log("\nThreshold check");
+  console.log('\nThreshold check');
```

ESLint

[error] Unexpected console statement.

(no-console)